### ISSUE TITLE
about sparse cnn quantize aware training

### ISSUE THREAD

**Issue by simonhandsome** [pytorch/pytorch#44880]
Hi, did anyone know that can sparse convolutional neural network being implemented in quantize aware training, cause if i implement my sparse cnn model into quantize aware training, it continuously appeared the error "max nan can't be less than min nan". 

cc @vincentqb @aocsa @nikitaved @pearu @jerryzh168 @jianyuh @dzhulgakov @raghuramank100 @jamesr66a @vkuzo

**Comment by nikitaved** [comment]
Could you please post the error message along with the reproduction script? To the best of my knowledge, it might be problematic, because autograd has a limitation when it comes to supporting dense grads for sparse inputs, and some functions handle sparse input but produce dense gradients.

**Comment by vkuzo** [comment]
closing, please feel free to reopen if this is still an issue
